src/codeu/chat/client/core/Context.java

```diff
@@ -39,7 +40,10 @@ public UserContext create(String name) {
         null :
         new UserContext(user, view, controller);
   }
-
+  //retrieves a serverinfo from the view object
```

Comment:
Prefer to write comments that are well-formatted English sentences, with a capitalized first letter and period at the end.

I don't want to spam the review by commenting on each one, but this applies elsewhere as well.